radiography/src/main/java/radiography/ViewFilters.kt

```diff
@@ -0,0 +1,50 @@
+package radiography
+
+import android.view.View
+
+public object ViewFilters {
+
+  /** A [ViewFilter] that matches everything. */
+  @JvmStatic
+  public val All: ViewFilter = viewFilterFor<Any> { true }
```

Comment:
Unfortunately not, because `Nothing` is not allowed to be passed for a reified type parameter. But you're right, this isn't as efficient as it could be. I'll make this an anonymous object instead.